rasa/core/brokers/pika.py

```diff
@@ -297,7 +297,7 @@ def _message(
     @rasa.shared.utils.common.lazy_property
     def rasa_environment(self) -> Optional[Text]:
         """Get value of the `RASA_ENVIRONMENT` environment variable."""
-        return os.environ.get("RASA_ENVIRONMENT")
+        return os.environ.get("RASA_ENVIRONMENT", "")
```

Comment:
@ePreda I've changed this section myself too with https://github.com/RasaHQ/rasa/pull/10614#discussion_r777035191 
The only difference is instead of empty string I've set it to a string which makes it clear that this env variable is missing, ie. not set.

Reply:
If you pull latest main, you should get my changes.

Reply:
Wonderful and explicit, but should have also changed this default in the pika event broker to maintain consistency. 

Will do it in this PR.